modules/csharp/pages/working-with-vector-search.adoc

```diff
@@ -154,69 +153,139 @@ include::csharp:example$code_snippets/VectorSearch.cs[tags=vs-create-predictive-
++
+. Set the vector from the computed vector derived from the updater value and your ML model.
+
+.. If there is no value for the vector, this will result in the underlying document to not be indexed.
+
+. Once all vectors have completed the update loop, finish updating.
+
+NOTE: `updater.finish()` will throw an error if any values inside the updater have not been set or skipped.
+
+== Vector Search SQL++ Support
+
+Couchbase Lite currently supports Hybrid Vector Search and the `APPROX_VECTOR_DISTANCE()` function.
+
+IMPORTANT: Similar to the xref:csharp:fts.adoc[Full Text Search] `match()` function, the `APPROX_VECTOR_DISTANCE()` function and Hybrid Vector Search cannot use the `OR` expression with the other expressions in the related `WHERE` clause.
+
+== Use Hybrid Vector Search
+
+You can use Hybrid Vector Search (Hybrid Search) to perform vector search in conjunction with regular SQL++ queries.
+With Hybrid Search, you perform vector search on documents that have already been filtered based on criteria specified in the `WHERE` clause.
 
-NOTE: The default value for the `limit` parameter is 3.
+NOTE: A `LIMIT` clause is required for non-hybrid Vector Search, this avoids a slow, exhaustive unlimited search of all possible vectors. 
 
-=== Use `vector_match()`
+=== Hybrid Vector Search with Full Text Match
+
+Below is an example of using Hybrid Search with the Full Text `match()` function.
 
 [source, csharp]
 ----
         
-include::csharp:example$code_snippets/VectorSearch.cs[tags=vs-use-vector-match]
+include::csharp:example$code_snippets/VectorSearch.cs[tags=vs-hybrid-ftmatch]
 
 ----
 
-This function performs vector search against a specific vector index identifier for the specified vector expression.
-If the specified index does not exist, an error will occur on creation of the query.
-The matched vectors will be returned up to the specified limit number, if the limit is not specified then the default value will be used. 
-The returned vectors are sorted by their distance values in ascending order by default.
+=== Prediction with Hybrid Vector Search
+
+Below is an example of using Hybrid Search with an array of vectors generated by the `Prediction()` function at index time.
+
+[source, csharp]
+----
+        
+include::csharp:example$code_snippets/VectorSearch.cs[tags=vs-hybrid-prediction]
 
-IMPORTANT: Similar to the xref:csharp:fts.adoc[Full Text Search] `match()` function, `vector_match()` can only be called alone or at the top level `AND` expression.
+----
 
-=== `vector_distance(vectorIndexIdentifier)`
+== `APPROX_VECTOR_DISTANCE(vector-expr, target-vector, [metric], [nprobes], [accurate])`
 
 [cols = "3*"]
 |===
 |Parameter |Is Required |Description
 
-|vectorIndexIdentifier
-|Required
-|The name of the vector index.
+|vector-expr
+|image:ROOT:yes.png[]
+|The expression returning a vector (NOT Index Name).
+Must match the expression specified in the vector index.
+|target-vector
```

Comment:
For all these tables, I would add a newline before each row, so you have some hope of figuring out what each cell is